admin/jobs/delete_expired_security_codes.py

```diff
@@ -8,7 +8,7 @@
 
 
 def delete_security_codes():
-    print(f'\nDeleting expired security codes', flush=True)
```

Comment:
The flush arg is important to make sure the buffered output is logged out to the console at the time of printing.